src/Components/Search/__tests__/SearchBar.test.tsx

```diff
@@ -0,0 +1,97 @@
+import Input from "Components/Input"
+import { SearchBarRefetchContainer as SearchBar } from "Components/Search/SearchBar"
+import { SuggestionItem } from "Components/Search/Suggestions/SuggestionItem"
+import { renderRelayTree } from "DevTools"
+import { MockBoot } from "DevTools/MockBoot"
+import { ReactWrapper } from "enzyme"
+import React from "react"
+import { graphql } from "react-relay"
+
+jest.unmock("react-relay")
+
+const searchResults = {
+  search: {
+    edges: [
+      {
+        node: {
+          displayLabel: "Percy Z",
+          href: "/cat/percy-z",
+          searchableType: "Cat",
+          id: "percy-z",
+        },
+      },
+    ],
+  },
+}
+
+const simulateTyping = (wrapper: ReactWrapper, text: string) => {
+  const textArea = wrapper.find("input")
+  textArea.simulate("focus")
+  // @ts-ignore
+  textArea.getDOMNode().value = text
+  textArea.simulate("change")
+}
+
+const getWrapper = (viewer, breakpoint = "xl") => {
+  return renderRelayTree({
+    Component: SearchBar,
+    query: graphql`
+      query SearchBarTestQuery($term: String!, $hasTerm: Boolean!) {
+        viewer {
+          ...SearchBar_viewer @arguments(term: $term, hasTerm: $hasTerm)
+        }
+      }
+    `,
+    mockData: {
+      viewer,
+    },
+    variables: {
+      term: "perc",
+      hasTerm: true,
+    },
+    wrapper: children => (
+      <MockBoot breakpoint={breakpoint as any}>{children}</MockBoot>
+    ),
+  })
+}
+
+describe("SearchBar", () => {
+  it("displays search results", async done => {
+    const component = await getWrapper(searchResults)
```

Comment:
I think it would be more clear to call this `wrapper` especially since the func is called `getWrapper`.